frontend/src/modules/Environments/components/EnvironmentTeams.js

```diff
@@ -325,6 +332,31 @@ export const EnvironmentTeams = ({ environment }) => {
     }
   };
 
+  const updateConsumptionRoleHandler = async (newRow) => {
+    const response = await client.mutate(
+      updateConsumptionRole({
```

Comment:
Are we validating the input parameters or ensuring that they are not empty? groupUri and consumptionRoleName cannot be empty